tests/publiccloud/img_proof.pm

```diff
@@ -105,6 +105,8 @@ sub run {
         }
         $instance->run_ssh_command(cmd => 'rpm -qa > /tmp/rpm_qa.txt', no_quote => 1);
         upload_logs('/tmp/rpm_qa.txt');
+        $instance->run_ssh_command(cmd => 'journalctl -b > /tmp/journalctl_b.txt', no_quote => 1);
```

Comment:
Does the `journalctl_b.txt` lands on the openQA VM? I cannot see that from here

Reply:
Yes, that is the reason for `, no_quote => 1`. I will include a VR for that.